docker/dockerfile.torch

```diff
@@ -7,9 +7,9 @@ ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/cuda/lib64:/usr/local/cuda/extra
 
 ENV DEBIAN_FRONTEND=noninteractive
 
-ARG RMM_VER=v0.21.06
-ARG CUDF_VER=v0.21.06
-ARG NVTAB_VER=v0.6
+ARG RMM_VER=v21.06.00
+ARG CUDF_VER=v21.06.01
+ARG NVTAB_VER=vnightly
```

Comment:
The other dockerfiles are on cudf 21.08 - do we want torch to be on 21.06?

Reply:
Nope good call.